can_tools/scrapers/official/GA/ga_vaccines.py

```diff
@@ -1,281 +1,133 @@
-            measurement="cumulative",
-            unit="people",
-            age="20-24",
-        ),
-        "25_34": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="25-34",
-        ),
-        "35_44": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="35-44",
-        ),
-        "45_54": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="45-54",
-        ),
-        "55_64": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="55-64",
-        ),
-        "65_74": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="65-74",
-        ),
-        "75_84": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="75-84",
-        ),
-        "85PLUS": CMU(
-            category="total_vaccine_initiated",
-            measurement="cumulative",
-            unit="people",
-            age="85_plus",
-        ),
+    demographic = "age"
+    sheet_name = "AGE_BY_COUNTY"
+    location_column = "COUNTYFIPS"
+    has_location = True
+    variables = {"PERSONVAX": variables.INITIATING_VACCINATIONS_ALL}
+    demographic_formatting = {
+        "00-05": "0-5",
+        "05_09": "5-9",
+        "10_14": "10-14",
+        "15_19": "15-19",
+        "20_24": "20-24",
+        "25_34": "25-34",
+        "35_44": "35-44",
+        "45_54": "45-54",
+        "55_64": "55-64",
+        "65_74": "65-74",
+        "75_84": "75-84",
+        "85PLUS": "85_plus",
     }
 
-    def fetch(self):
-        return self.get_all_jsons(self.service, self.sheet, "6")
-
-    def normalize(self, data):
-        df = self.arcgis_jsons_to_df(data)
-        df["COUNTS"] = pd.to_numeric(df["COUNTS"], errors="coerce")
-        df = (
-            df.pivot_table(
-                index="COUNTYFIPS", columns=self.column_names, values="COUNTS"
-            )
-            .reset_index()
-            .rename_axis(None, axis=1)
+    def normalize(self, data: requests.models.Response) -> pd.DataFrame:
+        sheet = pd.read_excel(data.content, sheet_name=self.sheet_name)
+        data = self._rename_or_add_date_and_location(
+            data=sheet,
+            location_column=self.location_column,
+            # Remove unwanted fips codes
+            # 0 = Georgia
+            # 99999 = Unknown
+            locations_to_drop=[0, 99999],
+            timezone="US/Eastern",
         )
-        df = self._rename_or_add_date_and_location(
-            df, location_column="COUNTYFIPS", timezone="US/Eastern"
+        return (
+            data.pipe(
```

Comment:
Mildly curious why you used `.pipe` instead of calling `_reshape_variables` directly here.  It would have made more sense to me if you chained the `pipe` off of the `_reanme_or_add_date_and_location()` return value.  Anyway, not a big deal. 😁

Reply:
Good point, there's no reason to use `pipe` here as far as I can tell, so I'm not sure why I did haha. I can update it if it's worth doing!

Reply:
Heh, nah, definitely not worth doing. 😁